csw-server/src/main/java/org/fao/geonet/kernel/csw/services/getrecords/CatalogSearcher.java

```diff
@@ -702,7 +704,16 @@ private void setGeom(Element element, Geometry fullGeom) throws IOException, JDO
         encoder.setOmitXMLDeclaration(true);
         encoder.setNamespaceAware(true);
 
+        // Force geometry to multi-polygon
+        if( fullGeom instanceof MultiPolygon) {
+            encoder.encode( (MultiPolygon) fullGeom, org.geotools.gml3.GML.MultiPolygon, out);
+        }
+        else {
+            Log.debug(Geonet.CSW_SEARCH, "Geometry " +fullGeom.getGeometryType()+" transformed to multi-polygon" );
+            encoder.encode(SpatialIndexWriter.toMultiPolygon(fullGeom), org.geotools.gml3.GML.MultiPolygon, out);
```

Comment:
Q: Is there a reason (part of the standard?) to force content to be multi-polygon here? Or could the original geometry be encoded...